ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/conversion_filters/109901_project-count-format.cy.js

```diff
@@ -0,0 +1,42 @@
+/// <reference types ='Cypress'/>
+
+Cypress._.each(['ipad-2'], (viewport) => {
```

Comment:
Why are we using ipad-2?

Reply:
Using ipad-2 mainly because I have heard the user are using small screen devices (like tablets) so thought this would be better resolution than the i-phone which were used in the previous tests :) 
I am going to confirm from Sasha/Dimple to make sure which type of devices users are using mostly and will try to use a resolution similar/close to that for cypress tests :)